### DISCUSSION TITLE
can't display Greek in epub

### DISCUSSION THREAD

**Discussion by Andley** [sumatrapdfreader/sumatrapdf#3263] [Q&A] [answered]
I find 3.4.6 can not display Greek text in epub. Can anyone help shed some light on this ?

p.s. alternatively, is there a way to change default font in eBook viewing mode ?

**Comment by GitHubRulesOK** [comment]
3.4.6 uses MuPDF engine for ebooks. Currently no means in SumatraPDF to alter the contents
always test current behaviour in pre-release as only that can be fixed https://www.sumatrapdfreader.org/prerelease

 Without a sample then have to say test with MuPDF-GL (currently 1.21.0 windows.zip https://mupdf.com/releases/index.html) if it fails with that 1.21 version then open a bugzilla report with them, if it works there but not SumatraPDF then supply sample here.

**Comment by BurkSchu** [comment]
I have an epub with greek letters. Normal greek letters are perfect, but greek letters with accent are not displayed. They are on the page "Greek extended" from Unicode. For example GREEK CAPITAL LETTER IOTA WITH PSILI, E1 BC B8. This is a problem with a German Tolino ereader, too. The problem is in the pre-release, too!

**Comment by kjk** [comment]
There's nothing I can do without access to file that shows the problem.

Most likely this kind of issue is mupdf issue and the best I can do is to report it to mupdf devs via: https://bugs.ghostscript.com/enter_bug.cgi?product=MuPDF

**Comment by GitHubRulesOK** [comment]
@kjk here is an example where accented greek letters are clearly not visible https://www.gutenberg.org/ebooks/39536.epub.noimages I think there were others in the past


![image](https://user-images.githubusercontent.com/12754913/217628739-1ae6133d-4b7f-4535-ac3b-c6c5a1416d61.png)

**Comment by kjk** [comment, marked as answer]
opened https://github.com/sumatrapdfreader/sumatrapdf/issues/3265 to track this

**Reply by Andley** [reply]
@kjk Thank you very much !!

**Comment by pstrg** [comment]
SumatraPDF 3.6.1
It seems that the solution adressed by #3265 still doesn't work well for epub.

<img width="493" height="83" alt="sumatrarendering" src="https://github.com/user-attachments/assets/6c603e41-2de4-4be1-9997-5dcb68a2ae01" />

The improperly rendered words in the image (both in parenthesis) are **ὁμώνυμα** and **συνώνυμα** meaning that at least _lowercase omicron with soft breathing_ and _omega with acute accent_ display as strange artifacts.

**Reply by GitHubRulesOK** [reply]
@pstrg Nothing could be done without a sample for analysis the above issue was fixed because the character font in use could be enhanced with noto so pre-release 3.7 or 3.6.1 have no issues with the sample provided
<img width="629" height="235" alt="image" src="https://github.com/user-attachments/assets/6f849625-8d8b-4839-8ca1-4b559952048a" />
<img width="661" height="263" alt="image" src="https://github.com/user-attachments/assets/af9ed8d7-8f54-409f-8400-efda46a5555b" />

**Reply by pstrg** [reply]
[SOLVED]
Sorry for reporting without deeper analysis.
It happens that the document itself specified inadequate fonts (with poor Unicode rendering).
Since it's ePub (not KFX, Amazon dearest) it's editable - modifying css resolved the issue.
Relief: SumatraPDF remains the best.
